exercises/practice/pop-count/.meta/config.json

```diff
@@ -0,0 +1,19 @@
+{
+  "authors": [
+    "kahgoh"
+  ],
```

Comment:
Since she suggested a new test, I would suggest to add @angelikatyborska to the list of contributors (currently empty therefore missing)

Reply:
It's fine, I have enough attributions on the Elixir track 😁 time to merge!